compiler/plugins/input/StableHLO/Conversion/test/vhlo_stablehlo_mix_invalid.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: iree-compile --compile-to=input --split-input-file %s
+
+vhlo.func_v1 @vhlo_stablehlo_func(%arg0: !vhlo.tensor_v1<!vhlo.i32_v1>) -> (!vhlo.tensor_v1<!vhlo.i32_v1>) {
+  // expected-error @+1 {{failed to materialize conversion for result #0 of operation 'vhlo.constant_v1' that remained live after conversion}}
+  %0 = "vhlo.constant_v1"() <{value = #vhlo.tensor_v1<dense<1> : tensor<i32>>}> : () -> !vhlo.tensor_v1<!vhlo.i32_v1>
+  // expected-note @+1 {{see existing live user here: %2 = "builtin.unrealized_conversion_cast"(%1) : (!vhlo.tensor_v1<!vhlo.i32_v1>) -> tensor<i32>}}
+  %1 = builtin.unrealized_conversion_cast %0 : !vhlo.tensor_v1<!vhlo.i32_v1> to tensor<i32>
+  %2 = stablehlo.abs %1 : tensor<i32>
```

Comment:
This isn't a very helpful error message. Are you sure that VHLO and StableHLO can't be mixed?

Reply:
+1, this is not the ideal error message from the perspective of end users. If this is truly not supported, I'd rather have a simple pass that detects this and issues a clear error, like I described in 2. in https://github.com/openxla/iree/pull/16999#discussion_r1568203814

Reply:
This error is coming from stablehlo package.

Reply:
This should be fixed as of the last 2 checkins.